### PR TITLE
docs(installation/cloudformation) guide for db-less mode

### PR DESCRIPTION
**Pull request by shashiranjan84**
### Summary

Add guide for stack creation in db-less mode

### Full changelog
* Updated installation guide for cloudformation

### CODE REVIEW

File: app/install/aws-cloudformation.md [hunk 1]
```diff
@@ -73,6 +75,36 @@ on AWS RDS for you.
 - [ap-southeast-2]({{ page.links.aws }}?region=ap-southeast-2#/stacks/new?stackName=kong-elb-postgres-pv&templateURL={{ page.links.templates.kong-postgres-pv }})
 - [sa-east-1]({{ page.links.aws }}?region=sa-east-1#/stacks/new?stackName=kong-elb-postgres-pv&templateURL={{ page.links.templates.kong-postgres-pv }})
 
+#### Kong in DB-less mode
+
+Provisions Kong resources in a new VPC or existing VPC. 
+
+Note: User would need to provide a S3 bucket location where `kong.yml` is
```
Comment (h3xar0n) [inline app/install/aws-cloudformation.md]:
```suggestion
Note: User would need to provide an S3 bucket location where `kong.yml` is
```

File: app/install/aws-cloudformation.md [hunk 2]
```diff
@@ -73,6 +75,36 @@ on AWS RDS for you.
...
+stored with declarative configuration to bootstrap all the Kong instances.
```
Comment (h3xar0n) [inline app/install/aws-cloudformation.md]:
```suggestion
stored with the declarative configuration to bootstrap all the Kong instances.
```

File: app/install/aws-cloudformation.md (hunk 1 above)
Comment (shashiranjan84) [reply app/install/aws-cloudformation.md]: Its not a vowel Why an?

Comment (levfishbluefish) [reply app/install/aws-cloudformation.md]: It's not actually vowel vs consonant but _vowel _sound__ vs _consonant _sound__ , S3 begins with a vowel sound

Comment (h3xar0n) [reply app/install/aws-cloudformation.md]: We say it as "an ess-three bucket."

https://docs.aws.amazon.com/AmazonS3/latest/user-guide/create-bucket.html

File: app/install/aws-cloudformation.md [hunk 3]
```diff
@@ -73,6 +75,36 @@ on AWS RDS for you.
...
 - [sa-east-1]({{ page.links.aws }}?region=sa-east-1#/stacks/new?stackName=kong-elb-postgres-pv&templateURL={{ page.links.templates.kong-postgres-pv }})
 
+#### Kong in DB-less mode
+
+Provisions Kong resources in a new VPC or existing VPC. 
+
+Note: User would need to provide an S3 bucket location where `kong.yml` is
+stored with the declarative configuration to bootstrap all the Kong instances.
+
+ ##### HVM AMI
```
Comment (levfishbluefish) [inline app/install/aws-cloudformation.md]:
```suggestion
##### HVM AMI
```

Comment (levfishbluefish) [reply app/install/aws-cloudformation.md]: This tab seems to be stopping markdown from rendering this as an h4

**Pull request merged**